matter/wifi/wf200/efr_spi.c

```diff
@@ -74,6 +74,10 @@ static bool spi_enabled = false;
 
 uint8_t wirq_irq_nb = SL_WFX_HOST_PINOUT_SPI_IRQ; // SL_WFX_HOST_PINOUT_SPI_WIRQ_PIN;
 
+#define LENGTH_0        	0
+#define PIN_OUT_SET	    	1
+#define PIN_OUT_CLEAR	    0
```

Comment:
align please. The restyle will take care of it but it must become an habit while you code

Reply:
done